pkgdb/include/flox/pkgdb/metrics.hh

```diff
@@ -0,0 +1,56 @@
+/* ========================================================================== *
+ *
+ * @file flox/pkgdb/metrics.hh
+ *
+ * @brief Metrics reporting
+ *
+ *
+ * -------------------------------------------------------------------------- */
+#include <sentry.h>
+#include <string>
+
+#pragma once
+
+namespace flox {
+
+class metricsReporting
```

Comment:
nit (naming/nb): Does cpp have conventions on capitalization of class names? I think start class names with a capital letter across the rest of pkgdb 🤷🏼‍♂️

Reply:
Yup, thanks!